src/style.css

```diff
@@ -1 +1,169 @@
-/* empty source */
+/* #region Normalization*/
+body {
+  margin: 0;
+  background: rgba(255, 255, 255, 1);
+}
+/*#endregion*/
+
+.mainCard {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  width: 200px;
+  border: 1px solid rgba(243, 243, 243, 1);
+  border-radius: 5px;
+
+}
+/* #region image */
+
+.image {
+  /* display: inline-block; */
+  margin: 32px 19px 36px;
+}
+

```

Comment:
if you give margin to `mainCard ` as suggested this whole `image` class div is not required